src/blueprints/tv2/value-objects/tv2-layers.ts

```diff
@@ -1,8 +1,15 @@
 // The enum values in this file is taken from Blueprints and still has to match those values until we control ingest.
+export enum Tv2VizLayer { // Todo: figure out if enum should have 'Viz' in name or not, as technically other graphics might need the same value.
```

Comment:
Are there any plans on when these todos should be done?

Reply:
Yes. it was meant to help me remeber to ask you and Lindved, which it now has done.
Should it have `Viz` as part of the name or not?

Reply:
`Viz` has been changed to `Graphic`.